src/rpc/blockchain.cpp

```diff
@@ -786,6 +786,90 @@ static RPCHelpMan getmempoolentry()
     };
 }
 
+static RPCHelpMan getmempooltxspendingprevout()
+{
+    return RPCHelpMan{"getmempooltxspendingprevout",
+                "\nScans the mempool to find transactions spending any of the given outputs\n",
```

Comment:
```suggestion
                "Scans the mempool to find transactions spending any of the given outputs",
```

nit: I think those can be removed

Reply:
Agreed, done in https://github.com/bitcoin/bitcoin/pull/24408/commits/4c96203ec36f5634d6f73f3b9a1d99bbd91dcc21